frame/evm/src/runner/stack.rs

```diff
@@ -827,6 +842,106 @@ where
 		self.substate
 			.recursive_is_cold(&|a: &Accessed| a.accessed_storage.contains(&(address, key)))
 	}
+
+	// TODO this impl behind a feature flag, otherwise no-op
+	fn record_external_opcode_cost(&mut self, opcode: Opcode, gas_cost: GasCost, target: evm::gasometer::StorageTarget) -> Result<(), ExitError> {
```

Comment:
Suggestion: define an associated table of type `[Option<Cost>, 256]` for all opcodes that have a static cost, like the evm does for gas

Reply:
Done, added temporary values for static opcodes. We will need to add functions to calculate ref_time for dynamic opcode costs as well.